services/clusters-keeper/src/simcore_service_clusters_keeper/data/prometheus.yml

```diff
@@ -0,0 +1,51 @@
+global:
+  scrape_interval: "30s"
```

Comment:
go for an uneven number, prime number, maybe 53 seconds or so. or 27. not 30 🗡️

Reply:
:+1:

Reply:
done